### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by dutterbutter**
## Motivation
- Setting executable permissions is also not required as the script does so.

### CODE REVIEW

**Comment by alexanderblv** [comment]
I think we need to request review and approve for merge from these people. 
cc/ @Karrq @Jrigada @nbaztec

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Follow these steps to quickly install the binaries for `foundry-zksync`:
    Begin by cloning the `foundry-zksync` repository from GitHub. This will download the latest version of the source code to your local machine.
 
    ```bash
-   git clone git@github.com:matter-labs/foundry-zksync.git
+   https://github.com/matter-labs/foundry-zksync.git
```
Comment (nbaztec) [inline README.md:71]: In general not an issue, just curious about the motivation to use https origin

**Review by nbaztec — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (alexanderblv) [reply README.md:71]: > In general not an issue, just curious about the motivation to use https origin

That's what it says in the main documentation https://docs.zksync.io/build/
https://github.com/matter-labs/zksync-web-era-docs

I just fixed a part of the code in line with it.

**Comment by alexanderblv** [comment]
@dutterbutter this PR is ready to merge.

File: README.md (hunk 1 above)
Comment (nbaztec) [reply README.md:71]: @alexanderblv hope you are aware that this is not your PR.

Comment (nbaztec) [inline README.md:71]: Seemingly pointed in https://github.com/matter-labs/foundry-zksync/issues/363#issuecomment-2114906440 there's no precedence to change this. The other changes look good though.

**Review by nbaztec — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alexanderblv) [reply README.md:71]: > @alexanderblv hope you are aware that this is not your PR.

I can't create a PR, but I created the issue that the part of PR was based on.

Comment (alexanderblv) [reply README.md:71]: > Seemingly pointed in [#363 (comment)](https://github.com/matter-labs/foundry-zksync/issues/363#issuecomment-2114906440) there's no precedence to change this. The other changes look good though.

We are talking about public access. In this case, the HTTPS link looks preferable

**Review by nbaztec — APPROVED** [review]

**Pull request merged**